src/obstacle_avoidance/test/obstacle_avoidance-test.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Created By: Gareth Ellis
+ * Created On: July 16th, 2016
+ * Description: Tests for MyNode
+ */
+
+#include <MyNode.h>
+#include <gtest/gtest.h>
+
+TEST(MyNode, firstTest) {
+  AvoidObstacleClass aoc;
+  EXPECT_EQ(0, aoc.getAngVelMsg());
+  EXPECT_EQ(LINEAR_VEL, aoc.getLinearVelMsg());
+  aoc.setLinearVelMsg(12);
+  aoc.setAngVelMsg(5);
+  EXPECT_EQ(12, aoc.getLinearVelMsg());
+  EXPECT_EQ(5, aoc.getAngVelMsg());
+}
```

Comment:
Not super important for this node, so I'm not too fussed about it, but any code merged into the actual code base would require significantly more unit testing.